forge/tests/error/main.rkt

```diff
@@ -56,7 +56,9 @@
     (list "example_impossible.frg" #rx"impossible")
     (list "failed_theorem.frg" #rx"failed.")
     (list "failed_unsat.frg" #rx"Failed test")
-    (list "failed_sat.frg" #rx"Failed test")
+    (list "failed_sat.frg" #rx"Failed test") 
+    (list "properties_undirected_tree_underconstraint_error.rkt" #rx"subproperty failed. Found instance")
+    (list "properties_undirected_tree_overconstraint_error.rkt" #rx"subproperty failed. Found instance")
```

Comment:
Ok, but `X-subproperty` is still pretty vague. It's hard to realize what failed (an auto-generated theorem) and why (bad implication).

Let's discuss later with Tim / playtesters.

Reply:
Agreed, but also the message should say what kind of property it is. (This is contingent on our surface syntax choice.)